frontend/constants/risks.ts

```diff
@@ -82,34 +82,34 @@ export const RISK_LEVELS_ICONS = {
 
 export const RISK_LEVELS_COLOURS = {
 	[LEVELS.UNSET]: {
-		color: COLOR.DARK_GRAY,
-		pinColor: PIN_COLORS.DARK_GRAY,
-		selectedColor: PIN_COLORS.DARK_GRAY
+		color: COLOR.SOFT_BLUE,
```

Comment:
@charence is there any reason why we need a color and pin color? they all seems to be the same.

Reply:
@carmenfan COLOR.DARK_GRAY is in hex `'#bfbfbf'` and PIN_COLORS.DARK_GRAY is parsed into an array of RGB values `[0.75,0.75,0.75]`.

I think this is done because HTML takes the hex value (perhaps it was easier to define the colour in hex as well?) and pin colour in unity takes the RGB value https://github.com/3drepo/3drepounity/blob/master/AssetBundleViewer/Assets/Utility/MessageParser.cs#L70

Reply:
Considering pin colour is derived from colour, i don't think it makes sense to have 3 entries like this. In fact, we should probably just have color and selected color in hex and whatever component is colouring the pin should just do a conversion.

But probably not worth investigating. Do the other change and this can be merged.